src/js/components/Grid/stories/typescript/App.tsx

```diff
@@ -58,7 +59,8 @@ export const AppGrid = () => {
       ) : (
         <Text>Grid is not supported by your browser</Text>
       )}
-    </Grommet>
+    </>
+    // </Grommet>
   );
 };
 
```

Comment:
Can we add the full prop to this story?
```suggestion
AppGrid.args = {
  full: true,
};

```